state/runtime/precompiled/precompiled.go

```diff
@@ -177,3 +197,14 @@ func extendByteSlice(b []byte, needLen int) []byte {
 
 	return b[:needLen]
 }
+
+// abiBoolMustEncode encodes the given value using the given ABI type.
+// panics if there is an error occurred.
+func abiBoolMustEncode(v bool, t *abi.Type) []byte {
```

Comment:
IDK if the second parameter makes sense, we can simply do `abi.MustNewType("bool")` as the declared var previously it's not used anywhere else.

Reply:
Fixed [97f3d78](https://github.com/0xPolygon/polygon-edge/pull/796/commits/97f3d78fdf9f276162c65df6b07b1d855b190e3d)